handler/handler.go

```diff
@@ -80,11 +88,26 @@ func (g *SpotHandler) Run(ctx context.Context) error {
 					// Stop after ACK.
 					t.Stop()
 				}
+
+				if !rebalanceRecommendationSent {
+					rebalanceRecommendation, err := g.metadataChecker.CheckRebalanceRecommendation(ctx)
+					if err != nil {
+						return err
```

Comment:
Why do we return in this method at all? maybe we should just log the error? We will retry after `pollWaitInterval`, =

Reply:
That's what is done when it's out of the function
```			if err != nil {
				g.log.Errorf("checking for cloud events: %v", err)
			}```

Reply:
Missed that :)